### DISCUSSION TITLE
Math support for Wiki-pages

### DISCUSSION THREAD

**Discussion by fuhto** [community/community#18605] [Other Feature Feedback, Questions, & Ideas]
Recently GitHub was introducing math support in markdown. It would be great if math expression would also be rendered on Wiki pages. It would help our team a lot to document our code.

**Comment by isinicolle** [comment]
De acuedo contigo

**Comment by nschloe** [comment]
Duplicate of https://github.com/github-community/community/discussions/17157.

**Comment by davidstaheli** [comment]
This is now supported in wikis: [Wikis now support math and Mermaid diagrams](https://github.blog/changelog/2022-08-09-wikis-now-support-math-and-mermaid-diagrams/). But please also check out @nschloe's excellent [https://github.com/nschloe/xhub](xhub) extension!
